mkosi/__init__.py

```diff
@@ -4368,24 +4368,37 @@ def generate_key_cert_pair(args: Args) -> None:
 
 
 def bump_image_version() -> None:
-    """Write current image version plus one to mkosi.version"""
-    version = Path("mkosi.version").read_text().strip()
-    v = version.split(".")
+    """
+    Read current version from 'mkosi.version' and run 'mkosi.bump'
+    or increase version plus one and write it back to 'mkosi.version'
+    """
 
-    try:
-        m = int(v[-1])
-    except ValueError:
-        new_version = version + ".2"
-        logging.info(
-            "Last component of current version is not a decimal integer, "
-            f"appending '.2', bumping '{version}' → '{new_version}'."
-        )
+    if not (version_file := Path("mkosi.version")).exists():
+        die(f"Cannot bump image version, '{version_file}' not found")
+
+    version = version_file.read_text().strip()
+
+    if (script := Path("mkosi.bump").absolute()).exists():
```

Comment:
All other scripts are configurable, so this should probably, too.

Reply:
To be fair, `mkosi.version` in this case also isn't currently configurable for this to work.

I could possibly fix that by introducing a `image_version_file` config option and making `image_version` depend on that. Then the parameter for the script would likely be `image_version` and would just be written to `image_version_file` without needing to re-read `image_version_file`. Basically:

```py
with open(config.image_version_file, "w") as o:
	run([config.bump_script, config.image_version], stdout=o, ...)
```

Does this make sense?

And under what section should `bump_script` go, `[Host]`?